### PR TITLE
guide to History Embedding with ChromaDB Added

### PR DESCRIPTION
**Pull request by DevGajjar28**
# Pull Request Template
#11 The Target audience is individual dev right, so I have designed for quick setup and minimal time wastage. It includes direct links and clear instructions to ensure even new developers can easily understand and implement the solution without unnecessary hassle. And also it followed by an template structure.
#claim11

## Writer's Checklist

- [x] **Follow Writing Structure**
  - [x] I have used more than two H2 subtitles to organize content.
  - [x] Include an introduction with at least two paragraphs before the first H2 section.
  - [x] Use bold, italic, underline, notes, quotes, TLDRs, and key points as needed.
  - [x] Include `code elements` where appropriate.
  - [x] Write the article in Markdown format.

- [x] **Fact-Check**
  - [x] I have verified all facts and data points included in the article.

- [ ] **Assets**
  - [ ] Save images in the `/assets` folder.
  - [ ] Follow naming conventions: `YYYYMMDD_title_of_the_article_IMG_NAME_NO.png`.

- [x] **Interlinking**
  - [x] Use `CTRL+F` to search for relevant keywords on:
    - [Daytona Dotfiles Sitemap](https://www.daytona.io/sitemap-dotfiles.xml)
    - [Daytona Definitions Sitemap](https://www.daytona.io/sitemap-definitions.xml)
  - [x] Interlink keywords using the appropriate URLs.

- [x] **Glossary/Definitions**
  - [x] Identify and mark new terms that could be added to the Definitions page.

- [x] **Review and Edit**
  - [x] Read the article out loud to catch any awkward phrasing.
  - [x] Run the draft through [Grammarly](https://grammarly.com) or a similar tool to correct grammar and spelling errors.
  - [x] Double-check adherence to the style guide and repository guidelines.

### CODE REVIEW

**Comment by giraffekey** [comment]
Lol this guy took my content and ran it through ChatGPT. It still says my name in the "author" field.

**Comment by DevGajjar28** [comment]
> Lol this guy took my content and ran it through ChatGPT. It still says my name in the "author" field.

I don't know that which name I have to put on author so I just saw and it's not your content , I have done by myself you. You can check the difference.

File: guides/Browser_History_Embedding_with_ChromaDB.md [hunk 1]
```diff
@@ -0,0 +1,242 @@
+---
+title: "Browser History Embedding and Search with ChromaDB"
+description: "A detailed guide on embedding and searching browser history using ChromaDB with support for both local and Azure embeddings."
+date: 2024-08-19
+author: "Jacob Gaffke"
```
Comment (nkkko) [inline guides/Browser_History_Embedding_with_ChromaDB.md:5]: You have literally copied @giraffekey PR and made some changes on it?

Comment (DevGajjar28) [reply guides/Browser_History_Embedding_with_ChromaDB.md:5]: check the content! I don't know who's name i have to write on author so i have used from giraffeket PR. That's all i have done from PR> can check the content both has different way to guide!

Comment (DevGajjar28) [reply guides/Browser_History_Embedding_with_ChromaDB.md:5]: @nkkko this is first time I am doing on this so i don't know who author to be! If it still look like copied please let me know i will withdraw PR.

Comment (DevGajjar28) [reply guides/Browser_History_Embedding_with_ChromaDB.md:5]: yes, I will keep that in mind. Thanks for guide.

**Pull request closed without merging**